<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formMainWindow_wl</class>
 <widget class="QMainWindow" name="formMainWindow_wl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">vokoscreenNG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidgetSideBar">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="../screencast.qrc">
         <normaloff>:/pictures/screencast/monitor-sidebar.png</normaloff>:/pictures/screencast/monitor-sidebar.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidgetScreencast">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/screencast/monitor.png</normaloff>:/pictures/screencast/monitor.png</iconset>
           </attribute>
           <attribute name="title">
            <string notr="true"/>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButtonScreencastFullscreen">
              <property name="text">
               <string>Fullscreen</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string notr="true">For full feature support please logout and start a X11 Desktop session.</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string notr="true">openSUSE Leap 15.3 KDE</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string notr="true">Fedora 34 KDE</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Fedora 34 Gnome</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string notr="true">openSUSE Tumbelweed KDE</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0" colspan="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string notr="true">Experimental Wayland support, successfully tested with</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="12">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="12">
             <widget class="QFrame" name="frame_information">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QGridLayout" name="gridLayout_6">
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>0</number>
                 </property>
                 <item row="2" column="0">
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="1" column="2">
                    <widget class="QLabel" name="labelVideoSize">
                     <property name="text">
                      <string notr="true">0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="3">
                    <widget class="QLabel" name="label_16">
                     <property name="text">
                      <string notr="true">KB</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>Video size:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="labelFreeSize">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string notr="true">0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QLabel" name="label_15">
                     <property name="text">
                      <string notr="true">MB</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>Free disk space:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1" colspan="3">
                    <widget class="Line" name="line_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="0" alignment="Qt::AlignHCenter">
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string>Information</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <spacer name="verticalSpacer_21">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="3" column="0" alignment="Qt::AlignHCenter">
                  <widget class="QLabel" name="label_Upate_tab_1">
                   <property name="text">
                    <string notr="true">TextLabel</string>
                   </property>
                   <property name="openExternalLinks">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="bottomMargin">
                    <number>3</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>Record Time:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="labelInfoRecordTime">
                     <property name="text">
                      <string notr="true">00:00:00</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="9" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radioButtonScreencastWindow">
              <property name="text">
               <string>Window</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/screencast/microphone.png</normaloff>:/pictures/screencast/microphone.png</iconset>
           </attribute>
           <attribute name="title">
            <string notr="true"/>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/screencast/format-codec.png</normaloff>:/pictures/screencast/format-codec.png</iconset>
           </attribute>
           <attribute name="title">
            <string notr="true"/>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="2">
               <widget class="QComboBox" name="comboBoxFormat"/>
              </item>
              <item row="2" column="2">
               <widget class="QComboBox" name="comboBoxVideoCodec"/>
              </item>
              <item row="0" column="1" colspan="3">
               <layout class="QHBoxLayout" name="horizontalLayout_slider_frames">
                <item>
                 <widget class="QToolButton" name="toolButtonFramesReset">
                  <property name="text">
                   <string notr="true"/>
                  </property>
                  <property name="icon">
                   <iconset resource="../screencast.qrc">
                    <normaloff>:/pictures/screencast/undo.png</normaloff>:/pictures/screencast/undo.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="labelFrames">
                  <property name="text">
                   <string>Frames</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="labelVideoCodec">
                  <property name="text">
                   <string>Videocodec</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="labelFormat">
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonStart">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>CTRL+SHIFT+F10</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>CTRL+SHIFT+F11</string>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>CTRL+SHIFT+F12</string>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonContinue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>CTRL+SHIFT+F12</string>
            </property>
            <property name="text">
             <string>Continue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPlay">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Play last record</string>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonScreencastOpenfolder">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Open folder</string>
            </property>
            <property name="text">
             <string>Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="../screencast.qrc">
         <normaloff>:/pictures/log/log-sidebar.png</normaloff>:/pictures/log/log-sidebar.png</iconset>
       </attribute>
       <attribute name="title">
        <string notr="true"/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextBrowser" name="textBrowserLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../screencast.qrc"/>
 </resources>
 <connections/>
</ui>
